test/test_individual.py

```diff
@@ -1,139 +1,165 @@
 import math
+import numpy as np
 import pickle
+from collections import namedtuple
+
 import pytest
 
 import cgp
-from cgp.individual import IndividualSingleGenome
+from cgp import IndividualSingleGenome, IndividualMultiGenome
 from cgp.genome import ID_INPUT_NODE, ID_OUTPUT_NODE, ID_NON_CODING_GENE
 
 
-def test_pickle_individual():
+TestParams = namedtuple("TestParams", ["genome_params", "primitives", "dna", "target_function"])
+params_list = [
+    TestParams(
+        genome_params={
+            "n_inputs": 1,
+            "n_outputs": 1,
+            "n_columns": 2,
+            "n_rows": 1,
+            "levels_back": 2,
+        },
+        primitives=(cgp.Add, cgp.Parameter),
+        dna=[
+            ID_INPUT_NODE,
+            ID_NON_CODING_GENE,
+            ID_NON_CODING_GENE,
+            1,
+            0,
+            0,
+            0,
+            0,
+            1,
+            ID_OUTPUT_NODE,
+            2,
+            ID_NON_CODING_GENE,
+        ],
+        target_function=lambda x, c: x + c,
+    )
+]
+FuncArgs = namedtuple("FuncArgs", ["x", "c"])
```

Comment:
(see above)
also, i'd rename this to something more specific like `graph_input_values`.

Reply:
Thanks, renamed. Note that namedtuples fall under classes when it comes to naming conventions, therefore `GraphInputValues`. 

Changed in 6cf310c53036ba0d3cb1057fc0a8d837bac938d8